pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -44,8 +44,9 @@ var NetworkInterfaceFactory = getNetworkClass
 var podInterfaceName = podInterface
 
 type PodCacheInterface struct {
-	Iface *v1.Interface `json:"iface,omitempty"`
-	PodIP string        `json:"podIP,omitempty"`
+	Iface  *v1.Interface `json:"iface,omitempty"`
+	PodIP  string        `json:"podIP,omitempty"`
+	PodIPs [2]string     `json:"podIPs,omitempty"`
```

Comment:
Same here, the size should be removed.

Reply:
its the one you suggested in the original suggestion , already done locally

there are more one place in the test you didnt shown yet i think, its known thanks i am on it.